lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_new_milestone_action.rb

```diff
@@ -11,6 +11,10 @@ def self.run(params)
 
         github_helper = Fastlane::Helper::GithubHelper.new(github_token: params[:github_token])
         last_stone = github_helper.get_last_milestone(repository)
+
+        UI.user_error!('No milestone found on the repository.') if last_stone.nil?
+        UI.user_error!("Milestone #{last_stone[:title]} has no due date.") if last_stone[:due_on].nil?
```

Comment:
Self-Review: I had mixed feelings about putting this validation here, however as those two parameters are required to execute the action and without them, we receive a generic runtime error, I decided to put this here. Let me know if you think this is overkill.